trio/_threads.py

```diff
@@ -404,12 +408,13 @@ def from_thread_run_sync(fn, *args, trio_token=None):
         RunFinishedError: if the corresponding call to `trio.run` has
             already completed.
         Cancelled: if the corresponding call to `trio.run` completes
```

Comment:
Not your fault, but this is copypasta from the async version -- sync functions can't raise Cancelled. So you can just remove the description of Cancelled as an exception-that-might-be-raised entirely.